docs/examples/events/prevent.py

```diff
@@ -0,0 +1,26 @@
+from textual.app import App, ComposeResult
+from textual.containers import Horizontal
+from textual.widgets import Button, Input
+
+
+class PreventApp(App):
+    """Demonstrates `prevent` context manager."""
+
+    def compose(self) -> ComposeResult:
+        yield Input()
+        yield Button("Clear", id="clear")
+
+    def on_button_pressed(self) -> None:
+        """Clear the text input."""
+        input = self.query_one(Input)
+        with input.prevent(Input.Changed):  # (1)
```

Comment:
```suggestion
        with input.prevent(Input.Changed):  # (1)!
```

Consider adding the `!` to make sure that that comment disappears when the code is pasted into the documentation.

Reply:
Didn't know that, thanks!